### PR TITLE
login and avatar added. refs #283

### PR DESCRIPTION
**Pull request by giteshk11**
# Issue Being Addressed

Put the issue(s) # that your PR is addressing here. e.g. #15, #29

#283 

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[ ] New Feature
[x] Update to Existing Feature
[ ] Other (state below)

# Description

**For Bug Fixes:** What was the root cause? How do your changes fix the bug effectively?

**For Refactors:** What are we refactoring and why? How do your changes address this need for refactoring?

**For New Features:** What feature are we adding? Explain your technical approach to doing this.

**For Feature Updates:** What feature are we updating? Explain your technical approach to doing this.

**For Other:** State what 'type' of PR yours is and elaborate on the PR's technical content.

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Log in as '...'
2. Go to '...'
3. Click on '...'
4. See the updated element x

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

**Comment by giteshk11** [comment]
@paulchrisluke i changed the redirection from settings to home now on logout.

File: src/assets/placeholder_avatar.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
...0WYKLDk1YcAbbd/uk5kFmsZ2SKcAonIPh/PksryW+zCFv5YPe7XwBc4B5ELZQpBZ9RLvYApECVN4K/c/RMwk4eyDSi7R5+InGaJxSoD2DWvqvU6/dNOC/QbNnLsOisVT2ttbBZrqJAnD2MKzLWDRVIe36RDdbQNY3mmoMLFYtn9upxZqKZPX6cHyf/pbklzlUzgAAILVT5G+t9Z3sRh9RWXCmAGqsAQtUpu0M7p9p6xsy0zGwAa1Uy9wNyKi86OaHoX0/UFvBLKqszLKBDmza/McgZi2Y+DReNJb6U140lo2zpXTWqNNBZw4w1gpYrNVHsAPLxqoBbdGqzQBmCs5euuED7D/eaK3256UdltZrE4Ap32Hro5D8Z1vFlkq1s52pbRLTMPlg1lk1YX0mH6wewpS1YArvmJKVm6KxxSI4olpwJ5suJOxIxkhhSs6bOZiyyTKplu7IWbAluBl8ZchSgJmWLy4uTFhuq5j23Nyc3HQtAxiHiooN5TU3b7pmTaaFg5LjJQOY9g2K3nIe1ObP8a5p46AiEoCtdXjPg6fUAT44YMX3IuUBzPtc6b1KwQGrlMPmQev0c5Xy2qCASRxwNCVEbLlTYJ1eT5KCoy2hExRBAcfiWLWDr+BwBQOs9MLJTq2z6PUKL7IMBthqL46icLPrQvfuCAbYele7oqBDd78LAhjnipBkVYQQZihnKwjgqkzPCtN0EMDWe1J2OvOE7GEZBPDOzk6Ue9924PGmt7e3Ox0XpVzvHbC1FkmlaNm5Rg1XiMoP74BxrnCyqiY4WThbvsU7YOstg7sFFKrVsHfAVfOgQ3vS3gHHmj3Ks+xQ2SXvgK01Gs0DV/TzUA1LE+CihHq8LgHuUYHqX0+A1Qn1eH+VAZycrB5HSodf974Gp21Sh4R6vDwB7lGBRb8eKvHvHXDofpNFgZR9Xag+l94Bp1Bl2UPn///PO2DVBt8/rfZQDcS9A47tmErRgRHqOIt3wLybkC45VRO69PBuQ9/iHTAnGTjkXTWhAwAnHXyLd8D03aA9Q9WENg/08/At3gEDF8hVE+CG6BifAHsaaQmwJ0WH+pnKAFZ/m9lPDYBQb0nzPkUnwD81hFr/X++Ak5OVAPvVgKdfq8wanCzY04j6/hnvU3RagxNgvxrw9GuV8aLTFO1pRH3/zG8QU8INN0rq+AAAAABJRU5ErkJggg=="/>
```
Comment (mort3za) [inline src/assets/placeholder_avatar.svg:7]: This is a PNG image with an SVG container, if you don't have SVG file, use PNG directly (ask @bamboochow to upload it to Cloudinary for you)

File: src/layouts/Default.vue [hunk 2]
```diff
@@ -24,16 +34,51 @@
...
+    performRedirect() {
+      this.$router.push({ name: this.isAuthenticated ? 'Account' : 'LogIn' });
+    }
   }
 };
 </script>
+
+<style scoped>
+.login-text {
+  color: #0e9ec2;
```
Comment (mort3za) [inline src/layouts/Default.vue:82]: You can use Tailwind classes instead. e.g. `text-button`  
Colors list is available in colors.js

File: src/assets/placeholder_avatar.svg (hunk 1 above)
Comment (giteshk11) [reply src/assets/placeholder_avatar.svg]: changed svg to png

File: src/layouts/Default.vue (hunk 2 above)
Comment (giteshk11) [reply src/layouts/Default.vue]: replaced login-text with text-button

**Pull request merged**